src/aria/pageEngine/utils/BaseNavigationManager.js

```diff
@@ -26,6 +26,8 @@ module.exports = Aria.classDefinition({
     $classpath : "aria.pageEngine.utils.BaseNavigationManager",
     $statics : {
 
+        STORAGE_NOT_UNAVAILABLE : "The storage is not available on this browser.",
```

Comment:
I would name this constant either `STORAGE_UNAVAILABLE` or `STORAGE_NOT_AVAILABLE` but `STORAGE_NOT_UNAVAILABLE` is really confusing!